src/main/resources/static/ui/app/index.html

```diff
@@ -4,7 +4,7 @@
 <!--[if IE 8]>         <html lang="en" class="no-js lt-ie9"> <![endif]-->
 <!--[if gt IE 8]><!--> <html lang="en" class="no-js"> <!--<![endif]-->
 <head>
-	<base href="/vireo/">
+	<base href="/">
```

Comment:
This is a perfect candidate for jsp-injected value... Are we sure we don't want to use jsp for the index.html instead?